### DISCUSSION TITLE
obj.update会还原自定义输入框中的输入的数据

### DISCUSSION THREAD

**Discussion by MR-Robot-Lee** [layui/layui#1567] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.3

### 问题描述

在table中通过templet模板自定义了数字输入框、备注和附件。

点击图标，弹窗填写附件和备注，在附件和备注弹窗中点击确认按钮关闭弹窗后，通过 `obj.update({desc: '这是备注'}, true);` 更新备注的自定义模板，备注的自定义模板可以正常更新。

但如果点击图标编辑备注前，在自定义数字输入框中填写了内容，通过 `obj.update({desc: '这是备注'}, true);` 更新数据后，自定义数字输入框中输入的内容，就会被还原到更新备注前的状态。这个问题该如何解决?

### 业务代码

```auto
layui.table.on('tool(tableId)', obj => {
    if (obj.event === 'edit-desc') {
        layer.prompt({title: '请输入备注'}, function(value, index, elem){
            layer.close(index);
            obj.update({ desc: value }, true);
      });
    }
})
```

### 截图补充

![PixPin_2024-01-13_15-55-11](https://github.com/layui/layui/assets/36070997/8d539227-228d-45e8-a6e8-c07cdba4ccbd)

### 浏览器

chrome最新版浏览器

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by bxjt123** [comment]
自定义的输入框应该是没做数据同步，输入框填写了内容实际表格里存的还是旧数据，可以在输入后更新表格数据/缓存，或者开启单元格编辑使用表格自带的输入框
